stablehlo/reference/Element.h

```diff
@@ -191,6 +191,9 @@ Element real(const Element &e);
 /// Returns the remainder for two Element objects.
 Element rem(const Element &e1, const Element &e2);
 
+/// Returns the value rounded to the nearest even of Element object.
```

Comment:
value rounded to nearest integer, breaking ties towards the even, of the Element object.